decidim-budgets/lib/decidim/budgets/seeds.rb

```diff
@@ -13,56 +13,76 @@ def initialize(participatory_space:)
       end
 
       def call
+        component = create_component!
+
+        rand(1...3).times do
+          create_budget!(component:)
+        end
+
+        Decidim::Budgets::Budget.where(component:).each do |budget|
+          rand(2...4).times do
+            project = create_project!(budget:)
+
+            create_attachments!(attached_to: project)
+
+            Decidim::Comments::Seed.comments_for(project)
+          end
+        end
+      end
+
+      def create_component!
         landing_page_content = Decidim::Faker::Localized.localized do
           "<h2>#{::Faker::Lorem.sentence}</h2>" \
             "<p>#{::Faker::Lorem.paragraph}</p>" \
             "<p>#{::Faker::Lorem.paragraph}</p>"
         end
 
-        component = Decidim::Component.create!(
+        Decidim::Component.create!(
           name: Decidim::Components::Namer.new(participatory_space.organization.available_locales, :budgets).i18n_name,
           manifest_name: :budgets,
           published_at: Time.current,
           participatory_space:,
           settings: {
+            geocoding_enabled: [true, false].sample,
             landing_page_content:,
             more_information_modal: Decidim::Faker::Localized.paragraph(sentence_count: 4),
             workflow: Decidim::Budgets.workflows.keys.sample
           }
         )
+      end
 
-        rand(1...3).times do
-          Decidim::Budgets::Budget.create!(
-            component:,
-            title: Decidim::Faker::Localized.sentence(word_count: 2),
-            description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
-              Decidim::Faker::Localized.paragraph(sentence_count: 3)
-            end,
-            total_budget: ::Faker::Number.number(digits: 8)
-          )
-        end
-
-        Decidim::Budgets::Budget.where(component:).each do |budget|
-          rand(2...4).times do
-            project = Decidim::Budgets::Project.create!(
-              budget:,
-              scope: participatory_space.organization.scopes.sample,
-              category: participatory_space.categories.sample,
-              title: Decidim::Faker::Localized.sentence(word_count: 2),
-              description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
-                Decidim::Faker::Localized.paragraph(sentence_count: 3)
-              end,
-              budget_amount: ::Faker::Number.between(from: Integer(budget.total_budget * 0.7), to: budget.total_budget)
-            )
+      def create_budget!(component:)
+        Decidim::Budgets::Budget.create!(
```

Comment:
There is no traceability added as we have for meetings, proposals and elections

Reply:
Done in [cd4694a](https://github.com/decidim/decidim/pull/12205/commits/cd4694aa8b6c452e8caf280953d668f6f9f62821)